http_15.go

```diff
@@ -0,0 +1,16 @@
+// +build !go1.6
```

Comment:
https://golang.org/pkg/go/build/

not sure, as long as tip reports a version higher than go1.6 we should be good.